client/templates/course/lesson/quiz/questions/autoform.js

```diff
@@ -0,0 +1,26 @@
+
+var addQuestionFormHooks = {
+    before: {
+        method: function(doc, params){
+            var editedQuestion = Session.get("currentQuestionToBuild");
+            editedQuestion.title = doc.title;
```

Comment:
AutoForm already handles this document properties, please stay closer to AutoForm when managing document properties.